download_and_create_reference_datasets/v02/hail_scripts/write_dataset_validation_ht.py

```diff
@@ -0,0 +1,53 @@
+import argparse as ap
+import logging
+
+import hail as hl
+from hail_scripts.v02.utils.computed_fields.vep import get_expr_for_vep_sorted_transcript_consequences_array, \
+    get_expr_for_worst_transcript_consequence_annotations_struct, CONSEQUENCE_TERM_RANK_LOOKUP
+
+logging.basicConfig(format='%(asctime)s %(levelname)-8s %(message)s')
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+p = ap.ArgumentParser()
+p.add_argument("-g", "--genome-version", help="Genome build: 37 or 38", choices=["37", "38"], default="37")
+args = p.parse_args()
+
+
+def read_gnomad_subset(genome_version):
+    logger.info("==> Read gnomAD subset")
+
+    # select a subset of gnomad genomes variants that are in > 90% of samples
+    ht = hl.split_multi(hl.read_table('gs://gnomad-public/release/2.1.1/ht/genomes/gnomad.genomes.r2.1.1.sites.ht'))
+    ht = hl.filter_intervals(ht, [hl.parse_locus_interval('1', reference_genome='GRCh%s'%genome_version)])
```

Comment:
Hail 0.2 validates loci, and GRCh38 contigs are prefixed with "chr". So for GRCh38, this would need to be `chr1` instead of `1`.

Reply:
Ahh that's right. I couldn't run it on 38 because it's not available yet so didn't test.